docs/build-instructions.md

```diff
@@ -39,44 +39,44 @@ Install [Docker CLI](https://www.docker.com/community-edition#/download) and a [
 
 Use the following commands to build and push your microservice containers.
 
-> **Note:** For the following steps, you can build and push the images by running
+> **Note:** For the following steps, you can build and push the images by running (requires running get_code.sh in this directory first.)
```

Comment:
maybe add this command into the code snippit?

Reply:
It was already described in a snippet earlier in the doc - so this is just a reminder.

Reply:
ah.  reads weird, but surely worth mentioning